packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -89,12 +89,16 @@ _The key elements of the component's public API surface:_
         -   x: number
         -   y: number
         -   value: string
-    -   `quadrant` - represents the orientation of the dies on the wafer map (the layout of the values on the dies). It can be represented by a const(as suggested [here](https://github.com/ni/nimble/blob/12a84ea7ad9103ab848aa2cd9f724e8853751a10/packages/nimble-components/docs/coding-conventions.md#use-const-objects-instead-of-typescript-enums)) with the following values:
+    -   `quadrant` - represents the direction of the two axes, X and Y, which are used for displaying the die grid on the wafer map. The four possible combinations are represented with the position of the quadrant respective to the specific axes directions . It can be represented by a const(as suggested [here](https://github.com/ni/nimble/blob/12a84ea7ad9103ab848aa2cd9f724e8853751a10/packages/nimble-components/docs/coding-conventions.md#use-const-objects-instead-of-typescript-enums)) with the following values:
     -   topLeft - ![Top Left Quadrant](./Resources/top_left.png)
     -   bottomLeft - ![Bottom Left Quadrant](./Resources/bottom_left.png)
     -   topRight - ![Top Right Quadrant](./Resources/top_right.png)
     -   bottomRight - ![Bottom Right Quadrant](./Resources/bottom_right.png)
-    -   `orientation` - represent the orientation of the notch on the wafer map outline (only visual). As only four static orientations are possible, it can be represented by an Enum with the following values: top, bottom, left, right.
+    -   `orientation` - represents the orientation of the notch on the wafer map outline. As only four static orientations are possible, it can be represented by an Enum with the following values: `top`, `bottom`, `left`, `right`. This value does not influence the die grid display in any measure, it affects only the circle outline of the wafer.
+    -   `dieGridOriginX` - represents the origin coordinates for the X axis of the dies grid for rendering the wafer map. Leaving the value `undefined` will set the value to the minimum X corner value of the bounding box of the dies coordinates.
```

Comment:
Yeah, I think there is missing context here. I would presume these four properties are meant to allow a user to provide a particular window into the data provided in the `dies` property. Is that accurate? If so, I would further guess the value in these APIs it that it allows the view to have access to all of the data, but allow a front-end app to adjust the window into that data, avoiding the need for the view to retrieve the data from a possible expensive network request again.

Even if I'm way off-base, it would be valuable to describe the reasons these properties are being provided.

Reply:
By my reading of the provided grid-dimensions document, `dieGridOriginX` and `dieGridOiginY` don't seem like they could be derived from `orientation`. I believe the way it's described, if all dies (provided to the `dies` property) had, for example, y-values between 3-5, you could provide a `dieGridOriginY` of 1, and a `dieGridRows` of 7, to get the dies to render in a constrained, centered vertical area in the circle of the wafer.

Is there a reason we can't make these attributes more semantically aligned? For instance, isn't another way to view `dieGridOriginY`, just `dieGridMinY`, and then `dieGridCols` just `dieGridMaxY`?

Reply:
Is your concrete proposal:
`dieGridMinX`, `dieGridMinY`, `dieGridMaxX`, `dieGridMaxY`
That sounds great to me!

I think we could even simplify at that point to:
`diesMinX`, `diesMinY`, `diesMaxX`, `diesMaxY`
I think that is nicely aligned with the dies X and Y properties they are constraining.

Reply:
Yes, the new API is meant to describe just the bounding box limits of the data input. Those are used when calculating the amount of vertical and horizontal space a single die will take based on the component dimensions.

They do not influence the orientation or the quadrant, the data will conform to those properties respectively when displayed.

The use-case we are going to apply the new API for is when we have partial data, but we want the display not to get stretched, just like having the full data. Another use case can be as mentioned reducing the bounding box if you want to 'zoom' in, but I do not know a situation when that would be needed.

I think it's a matter of choice in how we want to abstract a delimited area in API. In the database the actual dimensions are the Y-diameter, X-diameter, and the coordinates of the center. Here we chose to use the starting point and the width and height. Having the minimum and maximum coordinates is also a valid choice.Are there any

Reply:
@munteannatan We gave a concrete proposal but sounds like there is still open discussion. I'd recommend creating a meeting to resolve it quicker as the timezone difference will slow down the question-response cycle.